src/main/java/no/sikt/sws/models/gateway/AliasDto.java

```diff
@@ -0,0 +1,37 @@
+package no.sikt.sws.models.gateway;
```

Comment:
Should we move all theese to models.opensearch?
If so we should also ensure that the files located in the opensearch package should all belong there:
I found the following 2 which at least should be considered moved to itnernal:
OpenSearchCommandKind
WorkspaceResponse

Reply:
moved